fuzz/fuzz_targets/stream.rs

```diff
@@ -0,0 +1,138 @@
+#![no_main]
+use std::{
+    io,
+    num::NonZeroUsize,
+    pin::Pin,
+    task::{Context, Poll},
+};
+
+use futures::{stream::StreamExt, SinkExt};
+use libfuzzer_sys::fuzz_target;
+use tokio_websockets::{CloseCode, Config, Limits, Message};
+extern crate tokio_websockets;
+use std::convert::TryFrom;
+
+use arbitrary::Arbitrary;
+use tokio::io::{AsyncRead, AsyncWrite, ReadBuf};
```

Comment:
The import ordering is still completely different than in the rustfmt.toml. It also would be good if the CI step for rustfmt would catch this :wink:

Reply:
This was the output of `cargo fmt`. I think `extern crate` in the middle was resetting the ordering. Moving `extern crate` above the imports seems to have resolved it.

Reply:
Are you using nightly rustfmt? The rustfmt.toml uses some nightly-exclusive options to make the formatting more uniform

Reply:
I use `rustc 1.79.0-nightly (f9b161492 2024-04-19)`. Is there still a formatting problem?

Reply:
No, the formatting is fine now :)